### ISSUE TITLE
Add jaeger remote sampler to SDK configuration

### ISSUE THREAD

**Issue by lonewolf3739** [open-telemetry/opentelemetry-python-contrib#636]
As described in SDK Spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration

**Comment by aabmass** [comment]
Do we even have a Jaeger Remote Sampler implementation yet? I can't find one

**Comment by srikanthccv** [comment]
No, it's not implemented. I created this issue to implement it but haven't had time to work on it.
